### PR TITLE
Fix missing parenthesis in folding

### PR DESCRIPTION
**Pull request by fxmarty**
These missing parenthesis may be somewhere else as well.

### CODE REVIEW

**Comment by btgraham** [comment]
Re: https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html#torch.nn.BatchNorm1d
Are parenthesis really missing on L122? If the input and `bn.running_mean` are exactly zero, I think the output of the combined 'Linear+BatchNorm' operation should be exactly equal to ` bn.bias`. So I don't think we should divide`bn.bias` by `(bn.running_var + bn.eps)**0.5` when forming `m`??

**Comment by fxmarty** [comment]
@btgraham Right, I misread, my bad!
